tests/test_web_request.py

```diff
@@ -610,6 +610,26 @@ def test_clone_headers_dict() -> None:
     assert dict(result) == {'a': 'b', 'c': 'd'}
 
 
+async def test_multipart_formdata_file(protocol) -> None:
+    # Make sure file uploads work, even without a content type
+    payload = StreamReader(protocol, loop=asyncio.get_event_loop())
+    payload.feed_data(b"""-----------------------------326931944431359\r
+Content-Disposition: form-data; name="a_file"; filename="binary"\r
+\r
+\ff\r
+-----------------------------326931944431359--\r\n""")
+    content_type = "multipart/form-data; boundary="\
```

Comment:
Could you please avoid escaping EOL? Sometimes this causes unobvious bugs.
do smth like
```python
content_type = (
    'str1'
    'str2'
)
```

Reply:
Totally, I just copied the other test and adapted it. Fixed.